pdffigures.cpp

```diff
@@ -71,7 +76,7 @@ int main(int argc, char **argv) {
 
   int opt;
   int optionIndex;
-  while ((opt = getopt_long(argc, argv, "svfrmij:a:o:p:", long_options,
+  while ((opt = getopt_long(argc, argv, "svfrmicj:a:o:p:", long_options,
```

Comment:
There also needs to be another check in line 139 so "No output requested" is not returned incorrectly